docs/tutorial/GPU_check.ipynb

```diff
@@ -1,690 +1,892 @@
+   "execution_count": 16,
+   "id": "f7dbdfc0",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2023-04-13T17:18:56.506646Z",
+     "start_time": "2023-04-13T17:18:56.505645Z"
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "1"
       ]
      },
-     {
-      "cell_type": "code",
-      "execution_count": 19,
-      "id": "c8a443d6",
-      "metadata": {
-       "ExecuteTime": {
-        "start_time": "2023-04-13T17:18:56.523880Z",
-        "end_time": "2023-04-13T17:18:56.927007Z"
-       }
-      },
-      "outputs": [],
-      "source": [
-       "import secretflow"
+     "execution_count": 16,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "jax.device_count()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d3b1b71b",
+   "metadata": {},
+   "source": [
+    "### Check the number of JAX processes associated with the backend of jax"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "id": "c6847ebe",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2023-04-13T17:18:56.523880Z",
+     "start_time": "2023-04-13T17:18:56.505645Z"
+    }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "1"
       ]
-     }
-    ],
-    "metadata": {
-     "kernelspec": {
-      "display_name": "Python 3 (ipykernel)",
-      "language": "python",
-      "name": "python3"
      },
-     "language_info": {
-      "codemirror_mode": {
-       "name": "ipython",
-       "version": 3
-      },
-      "file_extension": ".py",
-      "mimetype": "text/x-python",
-      "name": "python",
-      "nbconvert_exporter": "python",
-      "pygments_lexer": "ipython3",
-      "version": "3.8.10"
-     }
-    },
-    "nbformat": 4,
-    "nbformat_minor": 5
-   }
-   
\ No newline at end of file
+     "execution_count": 17,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "jax.process_count()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6b89568d",
+   "metadata": {},
+   "source": [
+    "### Check the backend of jaxlib"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "id": "a490cc61",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2023-04-13T17:18:56.524833Z",
+     "start_time": "2023-04-13T17:18:56.523880Z"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "gpu\n"
+     ]
+    }
+   ],
+   "source": [
+    "from jax.lib import xla_bridge\n",
+    "print(xla_bridge.get_backend().platform)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "79e98d71",
+   "metadata": {},
+   "source": [
+    "### Run a linear regression demo to verify that the Jax can run well\n",
+    "The demo code is from [https://www.secretflow.org.cn/docs/secretflow/zh_CN/tutorial/lr_with_spu.html](https://www.secretflow.org.cn/docs/secretflow/zh_CN/tutorial/lr_with_spu.html)\n",
```

Comment:
The Eng version should use https://www.secretflow.org.cn/docs/secretflow/en/tutorial/lr_with_spu.html while the CN version in po file should be https://www.secretflow.org.cn/docs/secretflow/zh_CN/tutorial/lr_with_spu.html